### PR TITLE
Test Cardano translations with an empty UTxO

### LINKED ISSUE #212: Test cardano translations with empty UTxO tables
For completeness, we should test translations with the empty UTxO. See `Test.Consensus.Cardano.Translation`

### PR DESCRIPTION
**Pull request by jorisdral**
# Description

Resolves #212

### CODE REVIEW

File: ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/Translation.hs [hunk 1]
```diff
@@ -87,8 +87,10 @@ tests = testGroup "UpdateTablesOnEraTransition"
                           byronToShelleyLedgerStateTranslation
                           byronUtxosAreInsertsInShelleyUtxoDiff
                           (\st -> cover 50 (      nonEmptyUtxosByron st) "UTxO set is not empty"
-                                -- TODO: #4473 we should test with empyt UTxO!
-                                -- . cover 1  (not $ nonEmptyUtxosByron st) "UTxO set is empty"
+                                . cover 1  (not $ nonEmptyUtxosByron st) "UTxO set is empty"
```
Comment (jorisdral) [inline ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/Translation.hs:90]: Adding this makes the tests very slow in some cases, because the probability of generating an empty UTxO is still relatively low. Do we still want to add this `cover`?

Comment (jorisdral) [reply ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/Translation.hs]: Seems to be solved by requiring `0.1` coverage instead of `1`.
